cmd/options.go

```diff
@@ -254,11 +254,54 @@ func (c *snapshotterOptions) validate() error {
 	if err := c.compressionConfig.Validate(); err != nil {
 		return err
 	}
-
 	return c.etcdConnectionConfig.Validate()
 }
 
 // complete completes the config.
 func (c *snapshotterOptions) complete() {
 	c.snapstoreConfig.Complete()
 }
+
+type copierOptions struct {
+	sourceSnapStoreConfig *types.SnapstoreConfig
+	snapstoreConfig       *types.SnapstoreConfig
+	maxSnapshots          int
+	maxSnapshotAge        int
+}
+
+func newCopierOptions() *copierOptions {
+	sourceSnapStoreConfig := snapstore.NewSnapstoreConfig()
+	sourceSnapStoreConfig.IsSource = true
+	return &copierOptions{
+		sourceSnapStoreConfig: sourceSnapStoreConfig,
+		snapstoreConfig:       snapstore.NewSnapstoreConfig(),
+	}
+}
+
+func (c *copierOptions) addFlags(fs *flag.FlagSet) {
+	fs.IntVar(&c.maxSnapshots, "last-n-snapshots", -1, "copy the last N snapshots form the store sorted by date from newest to oldest")
+	fs.IntVar(&c.maxSnapshotAge, "max-age-days", -1, "copy the last snapshots form the store not older than N days")
```

Comment:
Let's think about a better and more consistent naming of these 2 parameters. My first proposal would be to name them after the variables, I like the naming of variables much better here, so `max-snapshots`, `max-snapshot-age`. @plkokanov any other suggestions? In any case, the naming of flags and their variable should be the same, the variables should match the flags as closely as possible.